tools/hot.js

```diff
@@ -0,0 +1,60 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+
+const WebSocket = require('ws');
+const chokidar = require('chokidar');
+
+var currentPath = process.cwd();
+const ws_url = 'ws://localhost:8765';
+
+const websocket = new WebSocket(ws_url);
+
+websocket.on('open', function() {
+  const watcher = chokidar.watch('.', {
+    ignored: /(^|[\/\\])\../,
+    persistent: true,
+    ignoreInitial: true,
+    awaitWriteFinish: {stabilityThreshold: 200, pollInterval: 100}
+  });
+  watcher.on('all', function(event, path) {
+    if (event === 'add' || event === 'change') {
+      if (path.endsWith('.qml')) {
+        const relPath = path.replace(/\\/g, '/');
+
+        const url = `file://${currentPath}/${relPath}`;
+        websocket.send(`live_reload ${url}`);
```

Comment:
We need unload_reload too.

Reply:
Added! I've added an unload command that will revert all things.